labelbox/schema/organization.py

```diff
@@ -42,6 +44,7 @@ def __init__(self, *args, **kwargs):
     users = Relationship.ToMany("User", False)
     projects = Relationship.ToMany("Project", True)
     webhooks = Relationship.ToMany("Webhook", False)
+    resourceTags = Relationship.ToMany("resource_tags", False)
```

Comment:
I think what @msokoloff1  was referring to was it should be resource_tags = ... since we use snake_case in the sdk for vars